.platform.app.yaml

```diff
@@ -63,6 +63,7 @@ hooks:
 
     poetry run python manage.py collectstatic --no-input
   deploy: |
+    poetry run python manage.py rename_app authors people
```

Comment:
Are we going to remove this after this change has gone live?

Reply:
Yeah, I think we will need to:
- Do a release to main
- Merge this code in to develop (and double check it works, I've tested on a spare env already though)
- Merge to main
- Delete this afterwards